lib/v8debugapi.js

```diff
@@ -267,7 +267,8 @@ module.exports.create = function(logger_, config_, fileStats_) {
     if (breakpoint.location.line >= fileStats[matchingScript].lines) {
       return setErrorStatusAndCallback(cb, breakpoint,
         StatusMessage.BREAKPOINT_SOURCE_LOCATION,
-        messages.INVALID_LINE_NUMBER);
+        messages.INVALID_LINE_NUMBER + matchingScript + ':' +
+          breakpoint.location.line);
```

Comment:
You should also indicate the number of lines that `matchingScript` has.

Maybe we should suggest that perhaps their source doesn't match the deployment image?